documentation/pin_modules.md

```diff
@@ -0,0 +1,39 @@
+# 🧪 Pin a module version
+
+**Experimental** Bolt version 2.30.0 introduced module workflows
+for Bolt projects that improve dependency management and make it easier to add
+new modules to your Bolt project. For more information, see [Module
+management](managing_modules.md).
+
+If you need to pin a module in your Bolt project to a specific version, you can
+add a version requirement to the module in your `bolt_project.yaml` and run
+`bolt module install --force`.
+
+**Before you begin:** [Opt in to experimental module
+workflows](managing_modules.md#opting_in).
+
+Follow these steps to pin a module version:
+1. Find the module under the `module` key in your `bolt_project.yaml` file.
+2. Add a `version_requirement` key that specifies the version requirements for
+   the module. For example, the following example from a `bolt_project.yaml`
+   file sets the Ruby Task Helper module to greater than or equal to 1.0.0, but
+   less than 3.0.0:
+   ```yaml
+   # bolt_project.yaml
+   ...
+   modules:
+   - name: puppetlabs/ruby_task_helper
+     version_requirement: '>= 1.0.0 < 3.0.0'
```

Comment:
```suggestion
   - name: puppetlabs/apache
     version_requirement: '>= 4.0.0 < 6.0.0'
```

`apache` is probably a better choice here since `ruby_task_helper` is unlikely to ever be a direct dependency of a project.